src/UI/images/Image.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styles from '../../styles.scss';
+import PropTypes from 'prop-types';
+
+function Image({ image, ...props }) {
+  return (
+    <div className={styles.images}>
+      <img src={image} {...props} />
```

Comment:
```suggestion
      <img src={image} alt={alt} {...props} />
```

Reply:
it's good that you made it reusable but go a step further and explicitly add it to the props to provide further clarity on how your component should be used